Условные операторы 4/Условные операторы 4/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Условные_операторы_4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            while (true)
+            {
+                Console.WriteLine("Введите значени х");
```

Comment:
Опечатка в сообщении
```suggestion
                Console.WriteLine("Введите значение х");
```